communication/slack-config/users.yaml

```diff
@@ -2,3 +2,12 @@
 # Slack user IDs.
 users:
   katharine: UBTBNJ6GL
+  jeefy: U5MCFK468
+  sarahnovotny: U0AGW7007
+  paris: U5SB22BBQ
+  castrojo: U1W1Q6PRQ
+  mrbobbytables: U511ZSKHD
+  alejandrox1: U6AS37R50
+  jdumars: U0YJS6LHL
+  jbeda: U09QZ63DX
+  ihor.dvoretskyi: U0CBHE6GM
```

Comment:
This should be `idvoretskyi`. (GitHub ID not Slack ID)

Reply:
Derp Fixed.

Reply:
Also if you could, sort alphabetically :x (Thinking future maintainability)